### ISSUE TITLE
Implement TASK_CARGO_TRANSPORT

### ISSUE THREAD

**Issue by Grey-Echo** [FlightControl-Master/MOOSE#353]
It will allow to create tasks for human players to transport cargo from one point to another. The task will work similar to TASK_CAS or TASK_BAI in that sense that it will use the same framework. It will be a state machine. 

Once defined in a MISSION, the task will appear as a menu item under the COMMAND CENTER -> MISSION -> CARGO menus. Once the player selects the task, he will be directed to a pickup location (where the cargo is). Once he lands, he can command the cargo to board into the carrier. Once the cargo is boarded, he can then takeoff and fly towards the deploy area (could be zone, or point, or anything else). Once he lands again, he can then deploy the cargo using the menu. Once the cargo is deployed, the task will go into Success, which is an event that can be captured by the MD to do stuff, like give extra scores or activate other groups or create new tasks etc.

**Comment by FlightControl-Master** [comment]
Thanks!

**Comment by FlightControl-Master** [comment]
Started with the development, those who wanna have a look, peek into the branch FlightControl-Task-Cargo-Transport ...
-- Created SET_CARGO
-- DATABASE handles CARGO
-- Event handles CARGO
-- Event triggers CARGO events
-- Menu system to pickup and deploy cargo
-- Menu system to board and unboard cargo
Test mission is TSK-100 ... dynamic loading of moose.

**Comment by FlightControl-Master** [comment]
Task TASK_CARGO_TRANSPORT has been created and is deployed and working. Closing this one. Any new issue to be reported as a separate one.
